### ISSUE TITLE
Delete trained data and sample images

### ISSUE THREAD

**Issue by MarkyVasconcelos** [Qualeams/Android-Face-Recognition-with-Deep-Learning-Test-Framework#39]
I took 5 photos while training but I'm required to have one.

Clearing the data cache from app settings and uninstalling plus reinstalling the app keep all trained data.

Don't it have a button for this? Where is this stored?

**Comment by sladomic** [comment]
The images are stored in the /sdcard/Pictures/facerecognition folder for easy access (so you can also add more images manually) https://github.com/Qualeams/Android-Face-Recognition-with-Deep-Learning-Test-Framework/blob/master/USER%20MANUAL.md#addperson

**Comment by MarkyVasconcelos** [comment]
Thanks, I made it with one image, but have to reopen the app to enable the "Recognition training" button.

**Comment by sladomic** [comment]
Yes, that's a bug. We can use this ticket to address this bug :)

**Comment by jocast80** [comment]
I found and delete the pictures but no training data was deleted. Is there a way to delete training faces?
